config/services_prod.yaml

```diff
@@ -0,0 +1,14 @@
+services:
+    # Is enabled in other environments through Twig.
+    # Exceptions are already handled by the ApiProblemBundle, but need to be logged.
```

Comment:
This is a pain. The ApiProblemBundle doesn't currently do any logging, if it did there would be double logging where Twig is enabled (eg dev).

Reply:
I thought Twig was a template engine, what's its interaction with exceptions and logging?

Reply:
[ExceptionListener](https://github.com/symfony/symfony/blob/b5c6892911fc7454a76528cb0d4d1af4336c7413/src/Symfony/Component/HttpKernel/EventListener/ExceptionListener.php) has two responsibilities for some reason: logging, and creating a response. [It's only enabled, however, by the TwigBundle](https://github.com/symfony/symfony/blob/b5c6892911fc7454a76528cb0d4d1af4336c7413/src/Symfony/Bundle/TwigBundle/Resources/config/twig.xml#L130-L136). We have the it enabled is `dev` as the dependency of the WebProfilerBundle, so see differing behaviour.

This all seems quite odd, so I'll search for/open an issue upstream.

Reply:
Related: https://github.com/symfony/symfony/issues/18552 and https://github.com/symfony/symfony/issues/28587.

Reply:
Tempted to say we should remove the listener being enabled through Twig and add logging to the ApiProblemBundle.

Reply:
> add logging to the ApiProblemBundle

It seems fair that if it is generating error responses it should also be capable of logging them. Maybe to depend on and enable logging, rather than implementing itself; but from the point of this project it's the same.